simbad/command_line/simbad_morda.py

```diff
@@ -54,7 +54,7 @@ def main():
         raise OSError("amore executable not found")
 
     gui = simbad.util.pyrvapi_results.SimbadOutput(
-        args.rvapi_document, args.webserver_uri, args.display_gui, log_file, args.work_dir
+        args.rvapi_document, args.webserver_uri, args.display_gui, log_file, args.work_dir, ccp4i2_xml=args.ccp4i2_xml
```

Comment:
Same as above, and probably in other scripts as well ... please adjust

Reply:
I've added in these changes